### PR TITLE
DTSPO-4415: Update to add required entry

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-4415

### Change description ###
Updated entry for `criticality` attribute to match environment name added in #19

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -18,22 +18,23 @@ locals {
...
+    demo      = "Medium"
+    dev       = "Low"
+    ptl       = "High"
+    preview   = "Low"
+    ldata     = "High"
+    sandbox   = "Low"
+    nle       = "High"
+    nonprod   = "Medium"
+    ptlsbox   = "Low"
+    sbox-init = "Low"
```
Comment (timja) [inline main.tf:37]: I don’t think init is right

Comment (madjava) [reply main.tf:37]: yeah, but didnt seem to have a choice as the env name is `sbox-init` had a look through some documentation as it's not a common sytax but thought to give it a go.

Comment (timja) [reply main.tf:37]:
```suggestion
    sbox-int = "Low"
```

Comment (madjava) [reply main.tf]: Thanks @timja did notice the typo. Updated.

**Review by timja — APPROVED** [review]

**Pull request merged**